ush/parsing_namelists_CICE.sh

```diff
@@ -105,7 +105,7 @@ cat > ice_in <<eof
    kdyn            = 1
    ndte            = 120
    revised_evp     = .false.
-   kevp_kernel     = 0
+   evp_algorithm   = 'standard_2d'
```

Comment:
&setup_nml
sec_init = @[SECS]
was added in tests/parm/ice_in_template but not here.

Reply:
> Looking at changes that were updated in ufs-weahter-model:  [ufs-community/ufs-weather-model@b26a896...develop](https://github.com/ufs-community/ufs-weather-model/compare/b26a896f2c9bada438414a51218bc72236925b8c...develop) I saw that ufs-weather-model modules are updated to use hpc stack 1.2.0 instead of 1.1.0 and that change is not reflected in the module files here.  I'm not sure if that's intentional or not?  In addition, the ESMF and FMS modules have been updated: module load esmf/8_2_0_beta_snapshot_14 module load fms/2021.03which is also not reflected in the module files in this update. 

The hpc-stack update seems to only be on wcoss2.

For esmf and fms, I'm not sure if it currently even matters, since it is correct at compile time (do we even need to be loading these at runtime?). But I will update them for consistency. In the slightly wider view: with the new dictates from NCO, module updates for ALL workflow components need to start being coordinated because NCO wants a single top-level versions file to control all modules.

> 
> Someone else would be a better person to review the updates related to the output frequency.
> 
> I'm still reviewing the namelist changes, do you have output from your run or should I just make a new run myself? I can tell the changes for MOM6 look okay, I've made one comment on CICE, but for input.nml - I'm still following all the diffs and it might be easier if there's output to just look at instead of just looking at the code.

/work/noaa/global/wckolczyn/noscrub/global-workflow/update_ufs
/work/noaa/global/wckolczyn/noscrub/global-workflow/update_ufs_atm

If you need the run directories, you'll have to rerun it, as I neglected to change KEEPDATA. I'm still following the directions I listed in PR #430 to run the coupled runs.

Reply:
Thanks Walter! I will look at your new output.